Gopkg.toml

```diff
@@ -132,3 +132,11 @@ required = [
   branch = "master"
   name = "github.com/metal3-io/baremetal-operator"
   source = "https://github.com/openshift/baremetal-operator"
+
+[[constraint]]
+  name = "github.com/ovirt/cluster-api-provider-ovirt"
+  branch = "v1alpha1"
+
+[[constraint]]
+  name = "github.com/ovirt/go-ovirt"
+  branch = "master"
```

Comment:
Also since this repo is not controlled in openshift, I would like to see it pinned to a better constraint than just the branch.

Reply:
done